src/views/lighttable.c

```diff
@@ -1264,7 +1264,7 @@ int expose_full_preview(dt_view_t *self, cairo_t *cr, int32_t width, int32_t hei
     dt_mipmap_cache_get(darktable.mipmap_cache, NULL, preload_stack[count], mip, DT_MIPMAP_PREFETCH, 'r');
 
   lib->image_over = DT_VIEW_DESERT;
-  cairo_set_source_rgb(cr, .1, .1, .1);
+  cairo_set_source_rgb(cr, 0.2, 0.2, 0.2);
```

Comment:
Please, I can understand that you prefer having them on top. We all have different views, but do not call the overlay hiding a kludge. Hiding overlays is something done since long time in many softwares.